src/api/controllers/problem.controller.ts

```diff
@@ -201,6 +205,9 @@ const updateProblem = async (
         statement: req.body.statement,
         title: req.body.title,
         hidden: req.body.hidden,
+        author: req.body.author,
+        difficulty: req.body.difficulty,
+        lastModified: Date(),
```

Comment:
do we need to add date created here?

Reply:
The date created never changes when a problem is update so it isn't needed to add here. If it was added here, the date created would become the date modified.